# UCS Virtual Machine Manager Daemon.
#
# Copyright 2010-2016 Univention GmbH
#
# http://www.univention.de/
#
# All rights reserved.
#
# The source code of the software contained in this package
# as well as the source package itself are made available
# under the terms of the GNU Affero General Public License version 3
# (GNU AGPL V3) as published by the Free Software Foundation.
#
# Binary versions of this package provided by Univention to you as
# well as other copyrighted, protected or trademarked materials like
# Logos, graphics, fonts, specific documentations and configurations,
# cryptographic keys etc. are subject to a license agreement between
# you and Univention.
#
# In the case you use the software under the terms of the GNU AGPL V3,
# the program is provided in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
# GNU Affero General Public License for more details.
#
# You should have received a copy of the GNU Affero General Public
# License with the Debian GNU/Linux or Univention distribution in file
# /usr/share/common-licenses/AGPL-3; if not, see
# <http://www.gnu.org/licenses/>.
#
#
# Philipp Hahn <hahn@univention.de>, 2010.
# Nicolas Ternisien <nicolas@bureau-cornavin.com>, 2016.
# Éric Bischoff <ebischoff@bureau-cornavin.com>, 2016.
msgid ""
msgstr ""
"Project-Id-Version: univention-virtual-machine-manager 0.9.333-1\n"
"Report-Msgid-Bugs-To: packages@univention.de\n"
"POT-Creation-Date: 2015-03-10 11:18+0100\n"
"PO-Revision-Date: 2016-07-03 21:15+0100\n"
"Last-Translator: Éric Bischoff <ebischoff@bureau-cornavin.com>\n"
"Language-Team: German <fr@bureau-cornavin.com>\n"
"Language: de\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Generator: Lokalize 2.0\n"

#: univention/uvmm/unix.py:256
msgid "Authentication failed"
msgstr "Échec de l'authentification"

#: univention/uvmm/unix.py:286
msgid "Authentication protocol violated"
msgstr "Protocole d'authentification violé"

#: univention/uvmm/commands.py:620
#, python-format
msgid "Command \"%(command)s\" is restricted"
msgstr "L'utilisation de la commande « %(command)s » est restreinte"

#: univention/uvmm/client.py:179 univention/uvmm/client.py:215
#, python-format
msgid "Could not connect to \"%(host)s:%(port)d\": %(errno)d"
msgstr "Impossible de se connecter à « %(host)s:%(port)d » : %(errno)d"

#: univention/uvmm/uvmm_ldap.py:186 univention/uvmm/uvmm_ldap.py:242
#: univention/uvmm/uvmm_ldap.py:275 univention/uvmm/uvmm_ldap.py:291
msgid "Could not open LDAP-Admin connection"
msgstr "Impossible d'ouvrir une connexion LDAP-Admin"

#: univention/uvmm/uvmm_ldap.py:168
msgid "Could not open LDAP-Machine connection"
msgstr "Impossible d'ouvrir une connexion LDAP-Machine"

#: univention/uvmm/client.py:125
#, python-format
msgid "Could not open socket \"%(path)s\": %(errno)d"
msgstr "Impossible d'ouvrir la socket « %(path)s » : %(errno)d"

#: univention/uvmm/uvmm_ldap.py:160
#, python-format
msgid "Could not query \"%(uri)s\""
msgstr "Impossible d'envoyer une requête à « %(uri)s »"

#: univention/uvmm/client.py:69
#, python-format
msgid "Could not send request: %(errno)d"
msgstr "Impossible d'envoyer la requête : %(errno)d"

#: univention/uvmm/node.py:1724
#, python-format
msgid "Domain \"%(domain)s\" already exists: %(uuid)s"
msgstr "Le domaine « %(domain)s » existe déjà : %(uuid)s"

#: univention/uvmm/node.py:1573
#, python-format
msgid "Domain \"%(domain)s\" in state \"%(state)s\" can not be migrated"
msgstr "Le domaine « %(domain)s » dans l'état « %(state)s » ne peut être migré"

#: univention/uvmm/node.py:1732
#, python-format
msgid "Domain \"%(domain)s\" is not shut off: %(state)d"
msgstr "Le domaine « %(domain)s » n'est pas éteint : %(state)d"

#: univention/uvmm/node.py:1246
#, python-format
msgid "Domain name \"%(domain)s\" already used by \"%(uuid)s\""
msgstr "Le nom de domaine « %(domain)s » est déjà utilisé par « %(uuid)s »"

#: univention/uvmm/client.py:100
msgid "EOS while waiting for answer."
msgstr "Signal « Fin de la connexion » reçu en attendant la réponse."

#: univention/uvmm/node.py:1905
#, python-format
msgid "Error cloning \"%(domain)s\": %(error)s"
msgstr "Erreur en dupliquant « %(domain)s » : %(error)s"

#: univention/uvmm/node.py:1613
#, python-format
msgid "Error creating \"%(domain)s\" snapshot: %(error)s"
msgstr "Erreur en créant un instantané de « %(domain)s » : %(error)s"

#: univention/uvmm/storage.py:93
#, python-format
msgid "Error creating storage pool \"%(pool)s\": %(error)s"
msgstr ""
"Erreur en créant l'espace de stockage mis en commun « %(pool)s » : %(error)s"

#: univention/uvmm/storage.py:273
#, python-format
msgid ""
"Error creating storage volume \"%(name)s\" for \"%(domain)s\": %(error)s"
msgstr ""
"Erreur en créant le volume de stockage « %(name)s » de « %"
"(domain)s » : %(error)s"

#: univention/uvmm/node.py:1338
#, python-format
msgid "Error defining domain \"%(domain)s\": %(error)s"
msgstr "Erreur en définissant le domaine « %(domain)s » : %(error)s"

#: univention/uvmm/node.py:1662 univention/uvmm/node.py:1671
#, python-format
msgid "Error deleting \"%(domain)s\" snapshot: %(error)s"
msgstr "Erreur en supprimant l'instantané « %(domain)s » : %(error)s"

#: univention/uvmm/storage.py:429 univention/uvmm/storage.py:452
#: univention/uvmm/storage.py:473
#, python-format
msgid "Error listing pools at \"%(uri)s\": %(error)s"
msgstr ""
"Erreur en obtenant la liste des espaces mis en commun de « %(uri)s » : "
"%(error)s"

#: univention/uvmm/storage.py:287 univention/uvmm/storage.py:307
#, python-format
msgid "Error listing volumes at \"%(uri)s\": %(error)s"
msgstr "Erreur en obtenant la liste des volumes de « %(uri)s » : %(error)s"

#: univention/uvmm/storage.py:145
#, python-format
msgid "Error locating storage pool \"%(pool)s\" for \"%(domain)s\": %(error)s"
msgstr ""
"Erreur en localisant l'espace de stockage mis en commun « %(pool)s » pour « %"
"(domain)s » : %(error)s"

#: univention/uvmm/storage.py:121 univention/uvmm/storage.py:183
#, python-format
msgid ""
"Error locating storage volume \"%(volume)s\" for \"%(domain)s\": %(error)s"
msgstr ""
"Erreur en localisant le volume de stockage « %(volume)s » pour « %"
"(domain)s » : %(error)s"

#: univention/uvmm/node.py:1472
#, python-format
msgid "Error managing domain \"%(domain)s\""
msgstr "Erreur en gérant le domaine « %(domain)s »"

#: univention/uvmm/node.py:1475 univention/uvmm/node.py:1478
#, python-format
msgid "Error managing domain \"%(domain)s\": %(error)s"
msgstr "Erreur en gérant le domaine « %(domain)s » : %(error)s"

#: univention/uvmm/node.py:1562 univention/uvmm/node.py:1592
#, python-format
msgid "Error migrating domain \"%(domain)s\": %(error)s"
msgstr "Erreur en migrant le domaine « %(domain)s » : %(error)s"

#: univention/uvmm/uvmm_ldap.py:125
#, python-format
msgid "Error reading %(file)s: %(msg)s"
msgstr "Erreur en lisant %(file)s : %(msg)s"

#: univention/uvmm/uvmm_ldap.py:128
#, python-format
msgid "Error reading incomplete %(file)s."
msgstr "Erreur en lisant le fichier incomplet %(file)s."

#: univention/uvmm/uvmm_ldap.py:100
#, python-format
msgid "Error removing %(file)s.old: %(msg)s"
msgstr "Erreur en supprimant %(file)s.old : %(msg)s"

#: univention/uvmm/node.py:1329
#, python-format
msgid "Error removing domain \"%(domain)s\": %(error)s"
msgstr "Erreur en supprimant le domaine « %(domain)s » : %(error)s"

#: univention/uvmm/uvmm_ldap.py:110
#, python-format
msgid "Error renaming %(file)s.new: %(msg)s"
msgstr "Erreur en renommant %(file)s.new : %(msg)s"

#: univention/uvmm/uvmm_ldap.py:105
#, python-format
msgid "Error renaming %(file)s: %(msg)s"
msgstr "Erreur en renommant %(file)s : %(msg)s"

#: univention/uvmm/node.py:1506
#, python-format
msgid "Error restoring domain \"%(domain)s\": %(error)s"
msgstr "Erreur en restaurant le domaine « %(domain)s » : %(error)s"

#: univention/uvmm/network.py:54
#, python-format
msgid "Error retrieving list of active networks: %(error)s"
msgstr "Erreur en obtenant la liste des réseaux actifs : %(error)s"

#: univention/uvmm/network.py:75
#, python-format
msgid "Error retrieving list of networks: %(error)s"
msgstr "Erreur en obtenant la liste des réseaux : %(error)s"

#: univention/uvmm/node.py:1251 univention/uvmm/node.py:1260
#, python-format
msgid "Error retrieving old domain \"%(domain)s\": %(error)s"
msgstr "Erreur en obtenant l'ancien domaine « %(domain)s » : %(error)s"

#: univention/uvmm/node.py:1639 univention/uvmm/node.py:1645
#, python-format
msgid "Error reverting \"%(domain)s\" to snapshot: %(error)s"
msgstr "Erreur en ramenant « %(domain)s » vers l'instantané : %(error)s"

#: univention/uvmm/node.py:1492
#, python-format
msgid "Error saving domain \"%(domain)s\": %(error)s"
msgstr "Erreur en enregistrant le domaine « %(domain)s » : %(error)s"

#: univention/uvmm/network.py:68
#, python-format
msgid "Error starting network %(name)s: %(error)s"
msgstr "Erreur en démarrant le réseau %(name)s : %(error)s"

#: univention/uvmm/node.py:1534
#, python-format
msgid "Error undefining domain \"%(domain)s\": %(error)s"
msgstr ""
"Erreur en supprimant la définition du domaine « %(domain)s » : %(error)s"

#: univention/uvmm/node.py:1689 univention/uvmm/node.py:1706
#, python-format
msgid "Error updating domain \"%(domain)s\""
msgstr "Erreur en actualisant le domaine « %(domain)s »"

#: univention/uvmm/node.py:1304 univention/uvmm/node.py:1307
#: univention/uvmm/node.py:1312
#, python-format
msgid "Error updating domain \"%(domain)s\": %(error)s"
msgstr "Erreur en actualisant le domaine « %(domain)s » : %(error)s"

#: univention/uvmm/client.py:110
#, python-format
msgid "Error while closing socket: %(errno)d"
msgstr "Erreur en fermant la socket : %(errno)d"

#: univention/uvmm/client.py:98
#, python-format
msgid "Error while waiting for answer: %(errno)d"
msgstr "Erreur en attendant la réponse : %(errno)d"

#: univention/uvmm/unix.py:159
#, python-format
msgid "Exception: %(exception)s"
msgstr "Exception : %(exception)s"

#: univention/uvmm/node.py:925
#, python-format
msgid "Failed to backup domain %(domain)s: %(xml)s"
msgstr "Échec de la sauvegarde du domaine %(domain)s : %(xml)s"

#: univention/uvmm/node.py:1878
#, python-format
msgid "Failed to clone volume \"%(volume)s\": %(error)s"
msgstr "Échec de la duplication du volume « %(volume)s » : %(error)s"

#: univention/uvmm/node.py:1843
#, python-format
msgid "Failed to generate new name for disk \"%(disk)s\""
msgstr "Échec de génération d'un nouveau nom pour le disque « %(disk)s »"

#: univention/uvmm/node.py:1298
msgid "Failed to update device."
msgstr "Échec de mise à jour du périphérique."

#: univention/uvmm/node.py:1357 univention/uvmm/node.py:1360
msgid "Failed to update the additionally information in the LDAP directory."
msgstr ""
"Échec d'actualisation des informations supplémentaires dans l'annuaire LDAP."

#: univention/uvmm/node.py:1711
#, python-format
msgid "Failto to update domain \"%(domain)s\""
msgstr "Échec d'actualisation du domaine « %(domain)s »"

#: univention/uvmm/node.py:860
#, python-format
msgid "Hypervisor \"%(uri)s\" is already connected."
msgstr "L'hyperviseur « %(uri)s » est déjà connecté."

#: univention/uvmm/node.py:872 univention/uvmm/node.py:881
#, python-format
msgid "Hypervisor \"%(uri)s\" is not connected."
msgstr "L'hyperviseur « %(uri)s » n'est pas connecté."

#: univention/uvmm/protocol.py:83
#, python-format
msgid "Incompatible version: %(major)d.%(minor)d"
msgstr "Version non compatible : %(major)d.%(minor)d"

#: univention/uvmm/uvmm_ldap.py:135
msgid "No SERVICES defined."
msgstr "Aucun SERVICE défini."

#: univention/uvmm/protocol.py:94
#, python-format
msgid "Not a Packet: %(type)s"
msgstr "N'est pas un paquet : %(type)s"

#: univention/uvmm/client.py:90
msgid "Not a UVMM_Response."
msgstr "N'est pas une réponse UVMM."

#: univention/uvmm/protocol.py:92
#, python-format
msgid "Not a valid Packet: %(msg)s"
msgstr "N'est pas un paquet valide : %(msg)s"

#: univention/uvmm/unix.py:112
#, python-format
msgid "Packet is no UVMM Request: %(type)s"
msgstr "Le paquet n'est pas une requête UVMM : %(type)s"

#: univention/uvmm/node.py:1599 univention/uvmm/node.py:1604
#: univention/uvmm/node.py:1620 univention/uvmm/node.py:1625
#: univention/uvmm/node.py:1652 univention/uvmm/node.py:1657
#, python-format
msgid "Snapshot not supported \"%(node)s\""
msgstr "L'instantané n'est pas pris en charge « %(node)s »"

#: univention/uvmm/ec2cloud.py:479
#, python-format
msgid ""
"The EC2 region returned an error for connection \"%(connection)s\":\n"
"\n"
"AuthFailure: The provided AWS access credentials could not be validated. "
"Please ensure that you are using the correct access keys. Consult the AWS "
"service documentation for details."
msgstr ""
"La région EC2 a retourné une erreur pour la connexion « %(connection)s » :\n"
"\n"
"AuthFailure : les preuves d'identité pour l'accès à AWS fournies n'ont pu être validées. "
"Veuillez vous assurer que vous utilisez des clés d'accès correctes. Consultez "
"la documentation du service AWS pour plus de détails."

#: univention/uvmm/ec2cloud.py:512
#, python-format
msgid ""
"The EC2 region returned an error for connection \"%(connection)s\":\n"
"\n"
"RequestExpired: Please check your system time to interact with AWS."
msgstr ""
"La région EC2 a retourné une erreur pour la connexion « %(connection)s » :\n"
"\n"
"RequestExpired: veuillez vérifier l'horloge de votre système pour interagir "
"avec AWS."

#: univention/uvmm/ec2cloud.py:514
#, python-format
msgid ""
"The EC2 region returned an error for connection \"%(connection)s\":\n"
"\n"
"UnauthorizedOperation: The provided AWS access credentials are not "
"authorized to perform this operation. Check your IAM policies, and ensure "
"that you are using the correct access keys. Also, the IAM user must have "
"appropriate access rights to interact with EC2, e.g. AmazonEC2FullAccess."
msgstr ""
"La région EC2 a retourné une erreur pour la connexion « %(connection)s » :\n"
"\n"
"UnauthorizedOperation : les preuves d'identité pour l'accès à AWS fournies "
"ne permettent pas d'effectuer cette opération. Vérifiez vos droits IAM et "
"assurez-vous que vous utilisez des clés d'accès correctes. De plus, "
"l'utilisateur IAM doit avoir des droits d'accès appropriés pour interagir "
"avec EC2, par ex. AmazonEC2FullAccess."

#: univention/uvmm/ec2cloud.py:510
#, python-format
msgid ""
"The EC2 region returned an error for connection \"%(connection)s\":\n"
"\n"
"Your AWS account is currently blocked. If you have questions, please contact "
"AWS Support."
msgstr ""
"La région EC2 a retourné une erreur pour la connexion « %(connection)s » :\n"
"\n"
"Votre compte AWS est actuellement bloqué. Si vous avez des questions, "
"veuillez contacter le service en ligne d'AWS."

#: univention/uvmm/client.py:108
msgid "Timed out while closing socket."
msgstr "Délai dépassé en fermant la connexion."

#: univention/uvmm/client.py:177 univention/uvmm/client.py:213
#, python-format
msgid "Timed out while connecting to \"%(host)s:%(port)d\"."
msgstr "Délai dépassé en se connectant à « %(host)s:%(port)d »."

#: univention/uvmm/client.py:123
#, python-format
msgid "Timed out while opening local socket \"%(path)s\"."
msgstr "Délai dépassé en ouvrant une connexion locale « %(path)s »."

#: univention/uvmm/client.py:96
msgid "Timed out while receiving data."
msgstr "Délai dépassé en recevant les données."

#: univention/uvmm/client.py:67
msgid "Timed out while sending data."
msgstr "Délai dépassé en envoyant les données."

#: univention/uvmm/client.py:159
#, python-format
msgid "Unknown PAM type: %(type)s"
msgstr "Type PAM inconnu : %(type)s"

#: univention/uvmm/commands.py:624
#, python-format
msgid "Unknown command \"%(command)s"
msgstr "Commande « %(command)s » inconnue"

#: univention/uvmm/node.py:404
#, python-format
msgid "Unknown disk type: %(type)d"
msgstr "Type de disque inconnu : %(type)d"

#: univention/uvmm/node.py:1108 univention/uvmm/node.py:1814
#, python-format
msgid "Unknown disk/type='%(type)s'"
msgstr "Type de disque inconnu='%(type)s'"

#: univention/uvmm/node.py:1389
#, python-format
msgid "Unknown domain \"%s\""
msgstr "Domaine inconnu « %s »"

#: univention/uvmm/node.py:1153
#, python-format
msgid "Unknown interface/type='%(type)s'"
msgstr "Type d'interface inconnu='%(type)s'"

#: univention/uvmm/node.py:1021
#, python-format
msgid "Unknown os/type='%(type)s'"
msgstr "Type de système d'exploitation inconnu='%(type)s'"

#: univention/uvmm/node.py:1435
#, python-format
msgid "Unsupported state transition %(cur_state)s to %(next_state)s"
msgstr ""
"Transition de l'état %(cur_state)s vers %(next_state)s non prise en charge"

#: univention/uvmm/storage.py:247
#, python-format
msgid "Unsupported storage-pool-type \"%(pool_type)s\" for \"%(domain)s\""
msgstr ""
"storage-pool-type « %(pool_type)s » non pris en charge pour « %(domain)s »"

#: univention/uvmm/node.py:1823
#, python-format
msgid "Volume \"%(volume)s\" not found: %(error)s"
msgstr "Volume « %(volume)s » non trouvé : %(error)s"

#: univention/uvmm/commands.py:61 univention/uvmm/commands.py:241
#, python-format
msgid "args != dict: %(args)s"
msgstr "args != dictionnaire : %(args)s"

#: univention/uvmm/commands.py:102 univention/uvmm/commands.py:132
#: univention/uvmm/commands.py:146 univention/uvmm/commands.py:159
#: univention/uvmm/commands.py:172 univention/uvmm/commands.py:185
#: univention/uvmm/commands.py:198 univention/uvmm/commands.py:211
#: univention/uvmm/commands.py:226 univention/uvmm/commands.py:239
#, python-format
msgid "conn_name != string: %(conn_name)s"
msgstr "conn_name != chaîne : %(conn_name)s"

#: univention/uvmm/commands.py:379
#, python-format
msgid "definition != Domain: %(domain)s"
msgstr "definition != domaine : %(domain)s"

#: univention/uvmm/commands.py:362 univention/uvmm/commands.py:396
#: univention/uvmm/commands.py:411 univention/uvmm/commands.py:426
#: univention/uvmm/commands.py:441 univention/uvmm/commands.py:462
#: univention/uvmm/commands.py:477 univention/uvmm/commands.py:492
#: univention/uvmm/commands.py:507 univention/uvmm/commands.py:520
#: univention/uvmm/commands.py:533
#, python-format
msgid "domain != string: %(domain)s"
msgstr "domain != chaîne : %(domain)s"

#: univention/uvmm/commands.py:118
#, python-format
msgid "freq != int: %(freq)s"
msgstr "freq != entier : %(freq)s"

#: univention/uvmm/commands.py:311
#, python-format
msgid "group != string: %(group)s"
msgstr "group != chaîne : %(group)s"

#: univention/uvmm/commands.py:298
#, python-format
msgid "hz != int: %(hz)s"
msgstr "hz != entier : %(hz)s"

#: univention/uvmm/commands.py:213 univention/uvmm/commands.py:228
#, python-format
msgid "instance_id != string: %(instance_id)s"
msgstr "instance_id != chaîne : %(instance_id)s"

#: univention/uvmm/commands.py:76 univention/uvmm/commands.py:120
#: univention/uvmm/commands.py:535
#, python-format
msgid "name != string: %(name)s"
msgstr "name != chaîne : %(name)s"

#: univention/uvmm/commands.py:89 univention/uvmm/commands.py:104
#: univention/uvmm/commands.py:313 univention/uvmm/commands.py:345
#, python-format
msgid "pattern != string: %(pattern)s"
msgstr "pattern != chaîne : %(pattern)s"

#: univention/uvmm/commands.py:479 univention/uvmm/commands.py:494
#: univention/uvmm/commands.py:509
#, python-format
msgid "snapshot != string: %(snapshot)s"
msgstr "snapshot != chaîne : %(snapshot)s"

#: univention/uvmm/commands.py:413 univention/uvmm/commands.py:428
#, python-format
msgid "statefile != string: %(file)s"
msgstr "statefile != chaîne : %(file)s"

#: univention/uvmm/commands.py:537
#, python-format
msgid "subst != dict: %(subst)s"
msgstr "subst != dictionnaire : %(subst)s"

#: univention/uvmm/commands.py:540 univention/uvmm/commands.py:542
#, python-format
msgid "subst[] != string: %(subst)s"
msgstr "subst[] != chaîne : %(subst)s"

#: univention/uvmm/commands.py:464
#, python-format
msgid "target_uri != string: %(uri)s"
msgstr "target_uri != chaîne : %(uri)s"

#: univention/uvmm/commands.py:64
#, python-format
msgid "testconnection is not a bool %(testconnection)s"
msgstr "testconnection n'est pas un booléen %(testconnection)s"

#: univention/uvmm/commands.py:281
#, python-format
msgid "unknown node %(uri)s"
msgstr "nœud inconnu %(uri)s"

#: univention/uvmm/commands.py:215 univention/uvmm/commands.py:398
#, python-format
msgid "unsupported state: %(state)s"
msgstr "état non pris en charge : %(state)s"

#: univention/uvmm/commands.py:251 univention/uvmm/commands.py:263
#: univention/uvmm/commands.py:275 univention/uvmm/commands.py:300
#: univention/uvmm/commands.py:343 univention/uvmm/commands.py:360
#: univention/uvmm/commands.py:377 univention/uvmm/commands.py:394
#: univention/uvmm/commands.py:409 univention/uvmm/commands.py:424
#: univention/uvmm/commands.py:439 univention/uvmm/commands.py:460
#: univention/uvmm/commands.py:475 univention/uvmm/commands.py:490
#: univention/uvmm/commands.py:505 univention/uvmm/commands.py:531
#: univention/uvmm/commands.py:557 univention/uvmm/commands.py:572
#: univention/uvmm/commands.py:589
#, python-format
msgid "uri != string: %(uri)s"
msgstr "uri != chaîne : %(uri)s"

#: univention/uvmm/commands.py:606
#, python-format
msgid "volume != string: %(volume)s"
msgstr "volume != chaîne : %(volume)s"

#: univention/uvmm/commands.py:443
#, python-format
msgid "volumes != list or None: %(volumes)s"
msgstr "volumes != liste ou None: %(volumes)s"

#: univention/uvmm/commands.py:447 univention/uvmm/commands.py:592
#, python-format
msgid "volumes[] != string: %(volume)s"
msgstr "volumes[] != chaîne : %(volume)s"

